internal/server/lifecycle/storage_bucket_backup.go

```diff
@@ -0,0 +1,31 @@
+package lifecycle
```

Comment:
I'd put this one into its own commit too `incusd/lifecycle: Add storage bucket backup events`